examples/factor_analysis/latent_factor_from_htfa.py

```diff
@@ -0,0 +1,85 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import os
+import sys
+import scipy.io
+import numpy as np
+import nibabel as nib
+from mpi4py import MPI
+from subprocess import call
+from scipy.stats import stats
+from nilearn.input_data import NiftiMasker
+
+comm = MPI.COMM_WORLD
+rank = comm.Get_rank()
+size = comm.Get_size()
+if rank == 0:
+    import logging
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+n_subj = 2
+data_dir = os.getcwd() + '/data'
+url = []
+url.append(' https://www.dropbox.com/s/r5s9tg4ekxzbrco/s0.mat?dl=0')
+url.append(' https://www.dropbox.com/s/39tr01m76vxwaqa/s1.mat?dl=0')
+
+#get fMRI data and scanner RAS coordinates
+data = []
+R = []
+for idx in range(n_subj):
+    if idx % size == rank:
+        #download data
+        file_name = os.path.join(data_dir, 's' + str(idx) + '.mat')
+        cmd = 'curl --location --create-dirs -o ' + file_name + url[idx]
+        try:
+            retcode = call(cmd, shell=True)
+            if retcode < 0:
+                print("File download was terminated by signal", -retcode, file=sys.stderr)
+            else:
+                print("Fiile download returned", retcode, file=sys.stderr)
+        except OSError as e:
+            print("File download failed:", e, file=sys.stderr)
+        all_data = scipy.io.loadmat(file_name)
+        bold = all_data['data']
+        # z-score the data
+        bold = stats.zscore(bold, axis=1, ddof=1)
+        data.append(bold)
+        R.append(all_data['R'])
+
+n_voxel, n_tr = data[0].shape
+
+# Run HTFA with downloaded data
+from brainiak.factor_analysis.htfa import HTFA
+help(HTFA)
```

Comment:
Let's skip printing the help in examples. It may work in a live demo (or even a notebook), but in a script it only makes it harder to follow the output.